view/frontend/web/js/view/payment/method-renderer/omise-offsite-alipay-method.js

```diff
@@ -44,64 +44,121 @@ define(
              *
              * @return {boolean}
              */
-            isActive: function() {
+            isActive: function () {
                 return true;
             },
 
+            /**
+             * Get Omise public key
+             *
+             * @return {string}
+             */
+            getPublicKey: function () {
+                return window.checkoutConfig.payment.omise_cc.publicKey;
+            },
+
+            /**
+             * Get order total amount
+             *
+             * @return {string}
+             */
+            getOrderAmount: function () {
+                return window.checkoutConfig.quoteData.grand_total * 100;
```

Comment:
not sure if that is the best way to get data about order

Reply:
Beware - not all currencies are `* 100` - JPY is `* 1`

Reply:
@jacekstan Magento treats `grand_total` and `grand_base_total` differently. Can you help make sure that this would work with multi-currency as well?